composer.json

```diff
@@ -102,10 +102,11 @@
         "drupal-composer/drupal-scaffold": "^2.6",
         "drupal-composer/preserve-paths": "^0.1.5",
         "drupal/admin_denied": "^1.0",
+        "drupal/components": "^1.1",
```

Comment:
@rupl This needs `composer install` . If there are issues (maybe caused by the theme branch below), delete the .lock file first.